README.md

```diff
@@ -231,6 +237,51 @@ By default, `beekeeper-scheduler-apiary` listens on port 8080, `beekeeper-path-c
 | `scheduler-delay-ms`                | No       | Amount of time (in milliseconds) between consecutive cleanups. Default value is `300000` (5 minutes after the previous cleanup completes). |
 | `Metastore-uri`                     | Yes      | URI of the Hive Metastore where tables to be cleaned-up are located. |
 
+## Beekeeper-API
+
+Beekeeper also has an API which provides access to the Beekeeper database and allows to see what metadata and paths are currently being managed.
+
+This allows the user to manually enter a database and a table name and check whether this table has been successfully registered in Beekeeper along with the TTL configuration: the current status of the table, the date and time it will be deleted, the current cleanup delay... etc.
```

Comment:
```suggestion
The user can manually enter a database and a table name and check whether this table has been successfully registered in Beekeeper along with the TTL configuration, the current status of the table, the date and time it will be deleted, the current cleanup delay... etc.
```